src/app/beer_garden/app.py

```diff
@@ -257,6 +268,8 @@ def _setup_events_manager(self):
 
         event_manager.register(QueueListener(action=local_callbacks))
         event_manager.register(QueueListener(action=downstream_callbacks))
+        event_manager.register(QueueListener(action=system_mapping_callback))
+        event_manager.register(QueueListener(action=garden_mapping_callback))
```

Comment:
Hmm, this makes me think we should talk about how to structure the callback listeners. I agree this is good for now because you want to apply the callback regardless of whether it's coming from the local garden or a downstream (right?)

Reply:
Yes, I want to handle it regardless of where it  is coming from. If a child forwards a system create, then I want to cache that and store it with the Garden object. We can sync about how we want to handle these callbacks. I think we could also drop the parent_http_processor.py module because you have it covered in your callback for events.